usaspending_api/etl/management/sql/c_file_linkage/C_to_D_Linkage.md

```diff
@@ -0,0 +1,69 @@
+# C to D Linkage
+
+### Scope = Website
+
+## Background
+
+File C (Award Financial) is submitted by agencies on a quartlery basis. Each record in File C represents an Award and its details, details which include an associated TAS.
+
+FPDS and FABS are pulled in nightly in the form of transactions. Transactions are grouped into awards based on the grouping of the following fields:
+* Contracts
+    * agency_id
+    * referenced_idv_agency_iden
+    * piid
+    * parent_award_id
+* Assistance
+    * awarding_sub_tier_agency_c
+    * fain
+    * uri
+
+
+File C records should be a subset of the Awards data ingested from FPDS/FABS. The linkage logic is defined below.
+
+## Linkage Logic
```

Comment:
@ross-williford @lpowell12 Can you please review the latest linkage logic documented here for approval?

Reply:
Approved by @lpowell12